File: ts/iq/key_transparency~get~multi_serialized_lookup+single_serialized_lookup.ts
/** <iq type="get" xmlns="key_transparency">. Replies with one of: Success, Error. */
export function buildKeyTransparencyMultiSerializedLookup(version?: string): BinaryNode {
  return { tag: 'iq', attrs: { xmlns: 'key_transparency', to: 's.whatsapp.net', type: 'get' }, content: [
      { tag: 'multi_serialized_lookup', attrs: { version: version } },
      { tag: 'single_serialized_lookup', attrs: {  } }, /* repeated */
    ] }
}

File: rust/iq/key_transparency~get~multi_serialized_lookup+single_serialized_lookup.rs
/// <iq type="get" xmlns="key_transparency">. Replies with one of: Success, Error.
pub fn build_key_transparency_multi_serialized_lookup(version: String) -> Node {
    NodeBuilder::new("iq")
        .attr("xmlns", "key_transparency")
        .attr("to", "s.whatsapp.net")
        .attr("type", "get")
        .child(NodeBuilder::new("multi_serialized_lookup").attr("version", version).build())
        .child(NodeBuilder::new("single_serialized_lookup").build()) // repeated
        .build()
}